microscope/testsuite/devices.py

```diff
@@ -250,6 +261,7 @@ def initialize(self):
         """
         _logger.info('Initializing.')
         time.sleep(0.5)
+        self._initialized = True
```

Comment:
Why not add this attribute during object construction? This would mean that one only neds to check for it's value and not have always also check if the attribute already exists.  Also, should it not be set back to `False` after shutdown?

Reply:
Perhaps not a good reason, but this would still work if a device fails to
set that member during __init__. In the TestCamera case, it's a simple
attribute. If we used a similar idea on real hardware, we might want some
other test (with library calls) to test that the devices is correctly
initialized, so _initialized could then be a property.

Reply:
This change is only being made to `TestCamera` so there is no what other device could do.

Reply:
Yes, because I've introduced it here for now, but we might want to move it over to the base Device class if it's useful.